src/reflection.jl

```diff
@@ -74,8 +74,15 @@ function find_callsites(CI::Union{Core.CodeInfo, IRCode}, stmt_info::Union{Vecto
                         # we ignore any implicit iterate calls.
                         return process_info(info.call)
                     elseif isa(info, ConstCallInfo)
-                        result = info.result
-                        return [ConstPropCallInfo(MICallInfo(result.linfo, rt), result)]
+                        if length(info.results) == 1
+                            result = info.results[1]
+                            @assert length(info.results) == 1
```

Comment:
```suggestion
```
The assert seems guaranteed to pass given the condition at the beginning of this block